ci_scripts/deploy_release.rb

```diff
@@ -137,7 +131,6 @@ def create_cleanup_pr
 
 steps = [
   method(:export_builds),
-  method(:changedoc_approve),
   method(:approve_pr),
   method(:create_docs_pr),
   method(:push_tag),
```

Comment:
You also need to remove the `changelog_done` step in line 142 (github won't let me leave a comment on that line)

Reply:
Thanks for catching this!